### PR TITLE
Updates for shorthand method literals syntax

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

**Comment by locks** [comment]
@michaelrkn confirm and merge?

File: source/getting-started/naming-conventions.md [hunk 1]
```diff
@@ -91,11 +91,13 @@ to the template.
...
 ```app/routes/favorites.js
+import ajax from 'ic-ajax';
+
 export default Ember.Route.extend({
-  model: function() {
+  model() {
     // the model is an Array of all of the posts
     // fetched from this url
-    return $.ajax('/a/service/url/where/posts/live');
+    return ajax('/a/service/url/where/posts/live');
```
Comment (michaelrkn) [inline source/getting-started/naming-conventions.md]: This is a thing I'm not familiar with - you don't need to prepend `$.` to `ajax`? What is it called on?

**Comment by michaelrkn** [comment]
wow! nice work!

File: source/getting-started/naming-conventions.md (hunk 1 above)
Comment (locks) [inline source/getting-started/naming-conventions.md]: see https://github.com/emberjs/guides/pull/462/files#diff-414c60298bd3dd9a7aa15654913a5bddR94

Comment (locks) [inline source/getting-started/naming-conventions.md]: I added the import since it's probably not changing anytime soon.

Comment (michaelrkn) [inline source/getting-started/naming-conventions.md]: ah missed that. thanks!

**Pull request merged**